src/database/seeds/Participants.ts

```diff
@@ -7,7 +7,7 @@ import { Participant, ParticipantStatus } from '../../api/entities/Participant';
 type ParticipantsType = ModelObject<Participant>;
 const sampleData: Optional<
```

Comment:
I wonder if we need to insert a user for the `AwaitingApproval example` participant. Otherwise, we will have a participantRequest without a user. I think this `const firstUser = participant.users.sort((a, b) => a.id - b.id)[0];` will break?

Reply:
Good catch - I just removed the awaiting approval seed and changed the participant for the user seed to be awaitingApproval. It was set to awaitingSigning but we're not using that at all at the moment.